src/bubbleSort.js

```diff
@@ -0,0 +1,13 @@
+export default function bubbleSort(arr){
+  for (var i = 0; i < arr.length; i++) {
+    for (var j = 0; j < arr.length; j++) {
```

Comment:
nested loops can often take additional time to complete. Try refactoring to minimize computations necessary to complete the function.